dashboard/Dockerfile

```diff
@@ -30,6 +30,21 @@ server {
     listen 80;
     root /usr/share/nginx/html;
     absolute_redirect off;
+
+    # Clickjacking protection, not supported via CSP meta.
+    add_header Content-Security-Policy "frame-ancestors 'none'" always;
+
+    # Full CSP via header, if not using the meta tag.
+    # Replace or remove the OIDC provider and institution logo URLs as needed.
+    # add_header Content-Security-Policy "default-src 'self'; script-src-attr 'none'; img-src 'self' data: http://institution.com/logo.png; frame-src 'none'; object-src 'none'; base-uri 'self'; form-action 'self'; connect-src 'self' http://oidcprovider.com/realms/example/; frame-ancestors 'none'" always;
+
+    # When serving over HTTPS:
```

Comment:
If using the "meta tag" and HTTPS does "upgrade-insecure-requests" still need to be added manually or will the CSP plugin handle that?